src/services/notification/Email/ReturnFromSuspensionEmail/send.js

```diff
@@ -2,13 +2,13 @@ const sendEmail = require("../sendEmail");
 const { FROM } = require("../../../../constants/email");
 
 
-const send = ({to}) => {
+const send = (userData,sessionData,classData) => {
   const msg = {
-    to,
+    to:userData.email,
```

Comment:
accept a separate parameter `to` which should accept email id. this will allow sending the email to multiple recipients.

e.g: `const send = (to, {userData, sessionData, classData})` => {}